File: packages/graph/__fixtures__/extractor-project/enums.py
from enum import Enum, StrEnum, IntEnum
from typing import Literal


class OrderStatus(StrEnum):
    PENDING = "pending"
    CONFIRMED = "confirmed"
    SHIPPED = "shipped"
    DELIVERED = "delivered"
    CANCELLED = "cancelled"


class Priority(IntEnum):
    LOW = 0
    MEDIUM = 1
    HIGH = 2
    CRITICAL = 3


UserRole = Literal["admin", "editor", "viewer", "guest"]


class PaymentMethod(Enum):
    CREDIT_CARD = "credit_card"
    DEBIT_CARD = "debit_card"
    PAYPAL = "paypal"
    BANK_TRANSFER = "bank_transfer"

File: packages/graph/__fixtures__/extractor-project/enums.ts
export enum OrderStatus {
  PENDING = 'pending',
  CONFIRMED = 'confirmed',
  SHIPPED = 'shipped',
  DELIVERED = 'delivered',
  CANCELLED = 'cancelled',
}

export const PaymentMethod = {
  CREDIT_CARD: 'credit_card',
  DEBIT_CARD: 'debit_card',
  PAYPAL: 'paypal',
  BANK_TRANSFER: 'bank_transfer',
} as const;

export type UserRole = 'admin' | 'editor' | 'viewer' | 'guest';

export enum Priority {
  LOW,
  MEDIUM,
  HIGH,
  CRITICAL,
}
